packages/components/src/DataList/DataList.test.tsx

```diff
@@ -96,6 +113,11 @@ describe("DataList", () => {
     it("should render the specified layout", () => {
       expect(screen.getAllByTestId(layoutWrapper)).toHaveLength(3);
       expect(screen.getAllByTestId(layoutItem)).toHaveLength(6);
+      BREAKPOINTS.forEach(size => {
+        expect(
+          getCssVarForListItem(`--data-list-${size}-display`, layoutWrapper),
+        ).toEqual(new Array(3).fill("block"));
```

Comment:
Probably going to change this to be the length of mock data. Same as below

Reply:
Actually because it renders the layout in the header as well. It would be mockData.length + 1. Not the biggest fan of that